src/index.html

```diff
@@ -5,10 +5,49 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img class="logo_img" src="./images/logo.png"
+        alt="Logo">
```

Comment:
<img width="278" alt="Screenshot 2023-07-10 at 14 40 16" src="https://github.com/mate-academy/layout_moyo-header/assets/135232001/08951dae-dee6-4b56-aa17-5066c65a8002">
Consider formatting your code accordingly to the code style guide